service/src/main/java/uk/gov/hmcts/reform/fpl/config/GatewayConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class GatewayConfiguration {
+    private final String gatewayUrl;
```

Comment:
Isn't it enough to name it url? it's obvious its gateway as we have it in the Class name :)